tests/test_nsec3.py

```diff
@@ -13,26 +13,27 @@
 # ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT
 # OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-from io import BytesIO
```

Comment:
Please squash this change into the previous commit, it is not necessary to pollute the history this way.